internal/eventgen/main.go

```diff
@@ -151,7 +154,8 @@ func main() {
 
 			items := strings.Split(pkey, "_")
 			for _, x := range items {
-				name += strings.Title(x)
+				caser := cases.Title(language.Und)
```

Comment:
I think we can safely assume the Asterisk developers to stay with English event names, so we should be able to use `language.English` here.
Also, we shouldn't need the caser particularly, so you could write this as:

```golang
name += cases.Title(language.English).String(x)
```